projects/knora-ui/src/lib/viewer/values/link-value/link-value.component.ts

```diff
@@ -0,0 +1,160 @@
+import {Component, Inject, Input, OnChanges, OnDestroy, OnInit, SimpleChanges} from '@angular/core';
+import {BaseValueComponent} from '../base-value.component';
+import {CreateLinkValue, ReadLinkValue, ReadResource, UpdateLinkValue, KnoraApiConnection} from '@knora/api';
+import {Subscription} from 'rxjs';
+import {AbstractControl, FormBuilder, FormControl, FormGroup, ValidatorFn} from '@angular/forms';
+import {KnoraApiConnectionToken} from '../../../core';
+
+export function resourceValidator(): ValidatorFn {
+  return (control: AbstractControl): { [key: string]: any } | null => {
+    const invalid = !(control.value instanceof ReadResource);
+    return invalid ? {'invalidType': {value: control.value}} : null;
+  };
+}
+
+@Component({
+  selector: 'kui-link-value',
+  templateUrl: './link-value.component.html',
+  styleUrls: ['./link-value.component.scss']
+})
+
+export class LinkValueComponent extends BaseValueComponent implements OnInit, OnChanges, OnDestroy {
+  @Input() displayValue?: ReadLinkValue;
+  @Input() parentResource: ReadResource;
+  @Input() propType: string;
```

Comment:
Could you rename this to property? It's just a property IRI, right?

Reply:
Even though I believe propType was clear, I changed it to propIri.

Reply:
Then probably we need to agree on a consistent terminology. Shall I make then issue for that so we can discuss that independently of this PR?